src/main/java/com/rackspace/ceres/app/model/PendingDownsampleSet.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.ceres.app.model;
+
+import java.time.Instant;
+import lombok.Data;
+
+@Data
+public class PendingDownsampleSet {
+  int partition;
+
+  Instant timeSlot;
+
+  String tenant;
```

Comment:
```suggestion
  String tenantId;
```

optional nit.  I don't know what you're using across the rest of the code but I do have a slight preference to this.

Reply:
I'll think about that change in a separate PR.